coordinator/tests/services/translation/request/dispense/dispense-notification.spec.ts

```diff
@@ -271,26 +250,7 @@ describe("fhir MedicationDispense maps correct values in DispenseNotificiation",
     })
   })
 
-  // eslint-disable-next-line max-len
-  test("performer.actor.(type === 'Practitioner') maps to pertinentInformation1.pertinentSupplyHeader.author.AgentPerson.agentPerson", () => {
-    medicationDispenses.forEach(medicationDispense => setPractitionerName(medicationDispense, "XX-TEST-VALUE"))
```

Comment:
This has been migrated to using unattended access user which is using ods lookup, we have tests surrounding lookups for ods already